test/unit/DetectChanges/BCBreak/PropertyBased/PropertyDocumentedTypeChangedTest.php

```diff
@@ -113,6 +113,13 @@ class TheClass {
      * @var GenericType<T1, T2>
      */ 
     public $propertyWithComplexDocblockThatCannotBeParsed;
+    
+    /**
+     * @var int
```

Comment:
This looks like a good/valid scenario: type changed when moving from docblock to native.

Do we have a scenario where the same happens, but the type didn't change?

Reply:
The type didn't change in this example, or am I misunderstanding you.

Reply:
Sorry, yes, I inverted it while writing the comment 🤦

What I basically mean is that we need both a positive and negative scenario for docblock -> native changes.

Reply:
Ok. Will remove the one that you think don't belong here, and add the second instead.